web/gui/dashboard_info.js

```diff
@@ -849,7 +849,37 @@ netdataDashboard.submenu = {
     'apps.net': {
         title: 'network',
         info: 'Netdata also gives a summary for eBPF charts in <a href="#menu_ip_submenu_kernel">Networking Stack submenu</a>.'
-    }
+    },
+
+    'system.ipc semaphores': {
+        info: 'System V semaphores is an inter-process communication (IPC) mechanism that widely available on UNIX systems. '+
+        'It allows processes or threads within a process to synchronize their actions. '+
+        'A semaphore is a value in a designated place in operating system (or kernel) storage that each process can check and then change. '+
+        'Depending on the value that is found, the process can use the resource or will find that it is already in use and must wait for some period before trying again. '+
+        'They are often used to monitor and control the availability of system resources such as shared memory segments. ' +
+        'For details, see <a href="https://man7.org/linux/man-pages/man7/svipc.7.html">svipc(7)</a> and '+
```

Comment:
Please, add `target="_blank"`for each URL to open them in a different tab/window.

Reply:
Indeed.